pkg/grpc/api.go

```diff
@@ -139,31 +129,32 @@ func NewAPI(opts APIOpts) API {
 			AppID:                opts.AppID,
 			Logger:               apiServerLogger,
 			Resiliency:           opts.Resiliency,
+			Actors:               opts.Actor,
 			CryptoProviders:      opts.CryptoProviders,
 			StateStores:          opts.StateStores,
 			SecretStores:         opts.SecretStores,
 			SecretsConfiguration: opts.SecretsConfiguration,
 			LockStores:           opts.LockStores,
 			WorkflowComponents:   opts.WorkflowComponents,
+
+			ShutdownFn:                 opts.Shutdown,
+			GetComponentsFn:            opts.GetComponentsFn,
+			GetComponentsCapabilitesFn: opts.GetComponentsCapabilitiesFn,
+			GetSubscriptionsFn:         opts.GetSubscriptionsFn,
 		},
-		directMessaging:            opts.DirectMessaging,
-		actor:                      opts.Actor,
-		id:                         opts.AppID,
-		resiliency:                 opts.Resiliency,
-		appChannel:                 opts.AppChannel,
-		pubsubAdapter:              opts.PubsubAdapter,
-		stateStores:                opts.StateStores,
-		configurationStores:        opts.ConfigurationStores,
-		configurationSubscribe:     make(map[string]chan struct{}),
-		sendToOutputBindingFn:      opts.SendToOutputBindingFn,
-		tracingSpec:                opts.TracingSpec,
-		accessControlList:          opts.AccessControlList,
-		appProtocol:                opts.AppProtocol,
-		shutdown:                   opts.Shutdown,
-		getComponentsFn:            opts.GetComponentsFn,
-		getComponentsCapabilitesFn: opts.GetComponentsCapabilitiesFn,
-		getSubscriptionsFn:         opts.GetSubscriptionsFn,
-		daprRunTimeVersion:         buildinfo.Version(),
+		directMessaging:        opts.DirectMessaging,
+		actor:                  opts.Actor,
+		id:                     opts.AppID,
```

Comment:
still `appid` to make it easier to understand?

Reply:
`id` is the current name. I would rather not change it in this PR, although I agree `appID` is better.

Reply:
I have made the change in this PR, at least for pkg/grpc/api